app/models/va_profile_redis/v2/contact_information.rb

```diff
@@ -179,6 +179,10 @@ def response_from_redis_or_service
       def contact_info_service
         @service ||= VAProfile::V2::ContactInformation::Service.new @user
       end
+
+      def verified_user?
+        @user.icn.present?
```

Comment:
What about the loa3 though? Does that not play a role anymore?

Reply:
ICN is the only thing neccessary for the upgrade Contact Information service. ICN == verified users